mythril/laser/ethereum/instructions.py

```diff
@@ -13,14 +13,17 @@
 import mythril.laser.ethereum.natives as natives
 from mythril.laser.ethereum.transaction import MessageCallTransaction, TransactionStartSignal, \
     ContractCreationTransaction
-from mythril.laser.ethereum.exceptions import VmException, StackUnderflowException
+from mythril.laser.ethereum.evm_exceptions import VmException, StackUnderflowException
 from mythril.laser.ethereum.keccak import KeccakFunctionManager
 
 TT256 = 2 ** 256
 TT256M1 = 2 ** 256 - 1
 
 keccak_function_manager = KeccakFunctionManager()
 
+class StopSignal(Exception):
```

Comment:
Is this being used?

Reply:
I don't think so, this is probably from before the TransactionEndSignal

Reply:
Confirmed, no current uses

Reply:
removed :+1: